frontend/src/app/projects/dashboard/page.tsx

```diff
@@ -0,0 +1,138 @@
+'use client'
+import { useQuery } from '@apollo/client'
+import {
+  faCheck,
+  faWarning,
+  faRectangleXmark,
+  faChartLine,
+  faUsers,
+  faStar,
+  faCodeBranch,
+  faChartColumn,
+  faHeart,
+} from '@fortawesome/free-solid-svg-icons'
+import millify from 'millify'
+import { useState, useEffect, FC } from 'react'
+import { handleAppError } from 'app/global-error'
+import { GET_PROJECT_HEALTH_STATS } from 'server/queries/projectsHealthDashboardQueries'
+import type { ProjectHealthStats } from 'types/projectHealthStats'
+import DashboardCard from 'components/DashboardCard'
+import DonutBarChart from 'components/DonutBarChart'
+import LineChart from 'components/LineChart'
+import LoadingSpinner from 'components/LoadingSpinner'
+import ProjectTypeDashboardCard from 'components/ProjectTypeDashboardCard'
+
+const ProjectsDashboardPage: FC = () => {
+  const [stats, setStats] = useState<ProjectHealthStats>()
+  const [isLoading, setIsLoading] = useState<boolean>(true)
+  const { data, error: graphQLRequestError } = useQuery(GET_PROJECT_HEALTH_STATS)
+
+  useEffect(() => {
+    if (data) {
+      setStats(data.projectHealthStats)
+      setIsLoading(false)
+    }
+    if (graphQLRequestError) {
+      handleAppError(graphQLRequestError)
+      setIsLoading(false)
+    }
+  }, [data, graphQLRequestError])
+
+  if (isLoading) {
+    return <LoadingSpinner />
+  }
+
+  if (!stats) {
+    return (
+      <div className="flex h-screen items-center justify-center">
+        <p className="text-lg text-gray-500">No project health data available</p>
+      </div>
+    )
+  }
+  const projectsCardsItems = [
+    {
+      type: 'healthy',
+      count: stats.projectsCountHealthy,
+      icon: faCheck,
+    },
+    {
+      type: 'needsAttention',
+      count: stats.projectsCountNeedAttention,
+      icon: faWarning,
+    },
+    {
+      type: 'unhealthy',
+      count: stats.projectsCountUnhealthy,
+      icon: faRectangleXmark,
+    },
+  ]
+  const dashboardCardsItems = [
+    {
+      title: 'Average Score',
+      icon: faChartLine,
+      stats: `${stats.averageScore.toFixed(1)}`,
+    },
+    {
+      title: 'Total Contributors',
+      icon: faUsers,
+      stats: millify(stats.totalContributors),
+    },
+    {
+      title: 'Total Forks',
```

Comment:
Same here. I think `total` can be omitted.
```suggestion
      title: 'Forks',
```